### PR TITLE
Updated contribute page to include hyperlinked images for list items …

### PR DESCRIPTION
**Pull request by JoshHeng**
Updated contribute page to include hyperlinked images for list items which illicited a hyperlink (e.g. to github, patreon, etc). Also sorted file identation

See issue https://github.com/CIrcuitVerse/CircuitVerse/issues/628 

# Screenshot
contribute-page-screenshot

### CODE REVIEW

**Comment by sakshi1499** [comment]
@tachyons It seems good to me. Should I approve it?

**Comment by tachyons** [comment]
Need to get reviewed by @nitin10s as he is making some changes on this page

**Comment by nitin10s** [comment]
Be specific with the links, I'll say. For instance: Find and report bugs should have the link - https://github.com/CircuitVerse/CircuitVerse/issues/new?assignees=&labels=&template=bug_report.md&title= 
and not the Github repository link.
The same goes for the other hyperlinks as well.

**Comment by sakshi1499** [comment]
Seems like a good idea. Where are we thinking to redirect in case of Bug Hunter? Also Do we have any invite friends option? If not we could add it in the new UX/UI? @nitin10s

**Comment by nitin10s** [comment]
Bug hunter would have the link as I mentioned previously: https://github.com/CircuitVerse/CircuitVerse/issues/new?assignees=&labels=&template=bug_report.md&title=

For Inviting, We can have such popups for users to mail their friends. But I guess, let's leave this as another potential task for GCI later. Whatsay @tachyons @sakshi1499 ?

test

**Comment by sakshi1499** [comment]
@nitin10s I think it looks good and well help promoting our platform and yes this should be completely different task. Here in this task, Josh can add the right and specific links.

**Comment by JoshHeng** [comment]
I've added the GitHub bug links mentioned but I can't think of any other links that can be more specific

**Comment by nitin10s** [comment]
Add new feature: https://github.com/CircuitVerse/CircuitVerse/issues/new?assignees=&labels=&template=feature_request.md&title=
FInd and fix bugs: https://github.com/CircuitVerse/CircuitVerse/issues

**Comment by JoshHeng** [comment]
Links added

**Comment by nitin10s** [comment]
@sakshi1499 Looks good to me.

**Comment by tachyons** [comment]
Looks good,

- Move images from public/img to app/assets/images and use rails image_tag
- Update target_branch to stories/new_ux

**Comment by sakshi1499** [comment]
Looks good. Just that we were discussing on the issue section that smaller icons will look better. It kind of looks a little clustered. @parikshit also suggested so, so if you can make the icons smaller and do as @tachyons said, it'll be perfect.

**Comment by JoshHeng** [comment]
firefox_DCe9EKVSvC

Changes made

File: app/views/logix/contribute.html.erb [hunk 1]
```diff
@@ -37,15 +37,15 @@
         <div class="card-body" style="background-color:#DCE3E3; ">
           <ul class="list-group" style="box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);" >
             <li class="list-group-item" >
-              <div style="display: grid; grid-template-columns: 1fr 3rem;">
+              <div style="display: grid; grid-template-columns: 1fr 2rem;">
                 <div>Create amazing circuits and share on the platform</div>
-                <div style="display: flex; align-items: center; margin-left: 0.2rem; margin-right: 0.1rem;"><a href="/simulator" target="_blank"><img src="/img/favicon.ico" alt="CircutVerse logo"/></a></div>
+                <div style="display: flex; align-items: center; margin-left: 0.2rem; margin-right: 0.1rem;"><a href="/simulator" target="_blank"><img src="/assets/SVGs/logo_final.svg" alt="CircutVerse logo"/></a></div>
```
Comment (tachyons) [inline app/views/logix/contribute.html.erb]: Use image tag helper, otherwise image link will break in production mode

**Comment by sakshi1499** [comment]
Even though it looks so amazing, we were targeting to keep the patreon logo change issue in a separate PR. Should we go ahead with that change in this PR only @tachyons . Great work though @JoshHeng Looks sooo nice.

**Comment by JoshHeng** [comment]
I can revert the Patreon logo if needed?
(Issue: https://github.com/CircuitVerse/CircuitVerse/issues/669 )

**Comment by sakshi1499** [comment]
Actually we were thinking of adding it in the new UI we are building but you added icons next to the link, so now we could either add the old icon everywhere or we can directly keep the new icon only. Lets wait for what @tachyons says and we'll follow that :)

**Comment by JoshHeng** [comment]
@sakshi1499 Are you referring to the Patreon logo or the new CircutVerse logo? Isn't the Patreon logo only on that page?

**Comment by sakshi1499** [comment]
I am referring to the Patreon logo and yes its only on this page. You added the updated icons in front of the links, so keeping the whole page with either the old logos be it in front of the links or the one at the bottom or keeping all the logos new, either of them is acceptable. Depends if we want to keep these new logos in the old UI or not. So that is what I am asking @tachyons

**Comment by sakshi1499** [comment]
@JoshHeng Great work! I'll approve your task now :)

**Comment by tachyons** [comment]
Great work @JoshHeng

**Pull request merged**